### ISSUE TITLE
Not finding present literal block in ==dev

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#617]
To test my two recently filed issues on the development version, I set tox to depend on `sphinx==dev` and got a whole new issue:

```
attest/deprecated.py:docstring of attest.deprecated.Assert.not_in:4: (WARNING/2) Literal block expected; none found.
```

The method in question is [[https://github.com/dag/attest/blob/87c674d64f79dcaa50eb53b67f2fa4e296329245/attest/deprecated.py#L194|here]] and looks like this:

```
#!python


def not_in(self, obj):
    """The negated form of :meth:`in_`, corresponding to the ``not in``
    operation::

         Assert(0).not_in([1, 2, 3])

     """
    return assert_(self.obj not in obj, '&#37;r in &#37;r' &#37; (self.obj, obj))
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/617
- Originally reported by: Dag Odenhall
- Originally created at: 2011-01-23T22:16:31.951

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-09-21 08:27:37+00:00_

Very strange... do you still see this?

**Comment by shimizukawa** [comment]
_From Bradley Ayers on 2012-04-19 06:48:50+00:00_

This problem still persists.

**Comment by shimizukawa** [comment]
_From Rob van der Linde on 2012-09-13 21:51:16+00:00_

I had the same problem with Sphinx 1.1.3, when I had a docstring comment similar to the bug description with one line of text followed by an indented literal block and then the end of the docstring.

I noticed however that the problem went away if I added another line at the bottom of the docstring, after the indented literal block.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2012-09-16 09:46:35+00:00_

Closes #617: Fix docstring preparation without included signature: only ignore indentation of one line, not two.

→ <<cset bf362e9ccbf03988116d1c55209ae17ec8c17117>>
